index.html

```diff
@@ -27,6 +41,7 @@
 	<div class="home-body">
 		
 		<h1 class="chloe-smart">Chloe Smart</h1>
+		<p class="software-developer">SoftwareDeveloper</p>
```

Comment:
on line 29, you define the div 'navbar' - but there is no styling. This lack of styling is impacting accessibility when the screen is made smaller